src/components/WelcomeIntro.js

```diff
@@ -1,13 +1,60 @@
 import React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import { Col, Container, Row, Button } from "react-bootstrap";
```

Comment:
I think this unused button import might be making the deployment tests fail (`'Button' is defined but never used`).  Unfortunately the tests treat all of the warnings like errors - and won't let you merge until you fix them 😦